### PR TITLE
Move `teleport integration configure` commands to their own file

### PR DESCRIPTION
**Pull request by marcoandredinis**
This PR moves all the `teleport integration configure` command handlers to their own file.

Commands always end in `return trace.Wrap(err)` without checking if the `err != nil`.

No other code changes were applied.

### CODE REVIEW

File: tool/teleport/common/integration_configure.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+	"github.com/gravitational/teleport/lib"
+	"github.com/gravitational/teleport/lib/config"
+	"github.com/gravitational/teleport/lib/integrations/awsoidc"
+	"github.com/gravitational/teleport/lib/integrations/externalauditstorage"
+	"github.com/gravitational/teleport/lib/integrations/externalauditstorage/easconfig"
+	"github.com/gravitational/teleport/lib/integrations/samlidp"
+	"github.com/gravitational/teleport/lib/integrations/samlidp/samlidpconfig"
+	"github.com/gravitational/teleport/lib/utils"
+)
+
```
Comment (codingllama) [inline tool/teleport/common/integration_configure.go:43]: Any code changes here besides adding the context to the functions?

**Review by codingllama — COMMENTED** [review]
Looks good.

**Review by greedy52 — APPROVED** [review]
> All the requests types were converted to pointers

Any benefit? Does the `req.CheckAndSetDefaults` checks for `nil` request?

**Review by codingllama — APPROVED** [review]
LGTM, but please see @greedy52's comment.

**Comment by marcoandredinis** [comment]
> > All the requests types were converted to pointers
> 
> Any benefit? Does the `req.CheckAndSetDefaults` checks for `nil` request?

I'll revert that part
I think it came because of a comment in the other PR, but it's only adding more failure modes.

**Review by codingllama — APPROVED** [review]

**Pull request merged**